staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -724,6 +724,12 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdownBasedOnPodPriority
 	// +optional
 	ShutdownGracePeriodByPodPriority []ShutdownGracePeriodByPodPriority `json:"shutdownGracePeriodByPodPriority,omitempty"`
+	// ShutdownSetPodsPhaseToTerminal sets pods to terminal phase during shutdown (enabled by default).
+	// If a node becomes ready again after graceful node shutdown (e.g. a reboot), then having ShutdownSetPodsPhaseToTerminal enabled, will result in pods not being started on the same node since the pods will be placed into terminal phase. In contrast, if ShutdownSetPodsPhaseToTerminal is disabled, pods will not be placed into terminal phase, and as result they will resume to run after the node becomes ready again.
```

Comment:
See comment above - please reflow the line.

Reply:
Done.